owner.txt

```diff
@@ -0,0 +1 @@
+3432808
```

Comment:
Is this a team or user?

Reply:
BridgeDownstream user. I've never used an owner.txt with a team ID before so I'm not sure how that behaves.